src/date-range-picker/calendar/grids/index.tsx

```diff
@@ -97,7 +80,7 @@ export const Grids = ({
       const newMonth = !isSingleGrid && direction === -1 ? addMonths(baseDate, -1) : baseDate;
       const nearestBaseDate = getBaseDate(newMonth, isDateEnabled);
 
-      const newFocusedDate = selectFocusedDate(focusedDate, nearestBaseDate, isDateEnabled);
+      const newFocusedDate = findDateToFocus(focusedDate, nearestBaseDate, isDateEnabled);
```

Comment:
If function does not make any side effects (actually selecting something), then it is find..., not select...